src/core/auth/pages/Login/Login.tsx

```diff
@@ -1,7 +1,7 @@
 import { useTranslation } from 'react-i18next';
 import LanguageSwitcher from '../../../components/LanguageSwitcher';
 import { Page } from '../../../components/Page/Page';
-import { LoginFormContainer } from '../../components/LoginFormContainer';
+import { LoginFormContainer } from '../../components/LoginForm/LoginFormContainer';
```

Comment:
We have a default export from the component, the import can be simplified:

`import LoginForm from '../../components/LoginForm';`

Reply:
Aknowledged and updated!